src/diffusers/models/transformers/transformer_qwenimage.py

```diff
@@ -340,6 +402,7 @@ def __call__(
             is_causal=False,
             backend=self._attention_backend,
             parallel_config=self._parallel_config,
+            seq_lens=joint_seq_lens,
```

Comment:
ohhh, instead of adding a new seq_lens argument to `dispatch_attnention_fn` do you think we can just prepare a `joint_attn_mask` that can be normalized by `_normalize_attn_mask` later?
https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/attention_dispatch.py#L540C1-L589C21